index.html

```diff
@@ -0,0 +1,352 @@
+        <a class="navbar-brand" href="./index.html">
+          <img src="./images/Enha (1).png" alt="logo" id="logoMain" />
+        </a>
+        <button
+          class="navbar-toggler main_btn"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#collapsibleNavbar"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div
+          class="collapse navbar-collapse justify-content-end"
+          id="collapsibleNavbar"
+        >
+          <ul id="menyOptions" class="navbar-nav nav2">
+            <li class="nav-item">
+              <a class="nav-link" href="./index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./pages/about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#spearkers">Speakers</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#partners">Partners</a>
+            </li>
+          </ul>
+          <ul id="popup" class="navbar-nav nav2">
+            <div id="cross-icon">
+              <!-- <img
+                onclick="off()"
+                class="style-li"
+                src="/images/close-Icon.png"
+                alt="close-Icon"
+              /> -->
+              <i
+                onclick="off()"
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                close
+              </i>
+            </div>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="./index.html">Home</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="./pages/about.html">About</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="#spearkers">Speakers</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="#partners">Partners</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- // Main Section // -->
+
+    <div id="main1" class="container-fluid">
+      <div class="container py-5 px-1">
+        <h2 class="mainText">"Hello, Sharing World"</h2>
+        <h1 class="mainText">
+          Enhance Your Skills
+          <br />
+          GLOBAL SUMMIT
+          <br />
+          2022
+        </h1>
+      </div>
+
+      <div class="container mainPara text-black my-3">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ad
+          laudantium voluptatum, modi natus perspiciatis. Lorem ipsum dolor sit
+          amet consectetur adipisicing elit.
+        </p>
+      </div>
+      <div class="container mt-3">
+        <h1>
+          2022.10.20(THU) ~ 21(FRI)
+        </h1>
```

Comment:
- [ ] Only the main heading for the the page should use the `h1` tag, subsequent tags should use h2, h3 e.t.c. for semantic purposes kindly fix this.

Reply:
- [ ] I think here you didn't change the heading since it is very important to make your website follow the best practices for semantic reasons so as requested by the previous reviewer please change it to have just one `h1` per page 🔥💯✨